core/src/main/java/com/datastax/oss/driver/api/core/config/DriverConfigProfile.java

```diff
@@ -61,8 +60,4 @@
   Duration getDuration(DriverOption option);
 
   DriverConfigProfile withDuration(DriverOption option, Duration value);
-
-  ConsistencyLevel getConsistencyLevel(DriverOption option);
```

Comment:
Did you decide these that these didn't merit having their own methods for resolving CL by option?  It looks like the changes don't require removing this, but having this here might have just been considered extraneous.

Reply:
Parsing CLs now requires `ConsistencyLevelRegistry`, which is in the context. The config can't have a reference to the context because it's created first.

But yeah apart from the technical difficulties it also feels extraneous to have a special case for CLs.